java/test/jmri/managers/DefaultSignalSystemManagerTest.java

```diff
@@ -48,6 +52,13 @@ public void testLoadBasicAspects() {
     @Test
     public void testLoad() {
         DefaultSignalSystemManager d = new DefaultSignalSystemManager();
+
+        // Remove all beans in the manager
+        Set<SignalSystem> set = new HashSet<>(d.getNamedBeanSet());
```

Comment:
This test should work without modification based on your changes in the non-test code, so this cleaning of a new object’s default state should not be required in the test.

Reply:
If I don't deregister the beans, I get the error:
`systemName is already registered: AAR-1946`